spec/offchain.tex

```diff
@@ -110,26 +112,40 @@ \subsection{Variables}
 \begin{itemize}
 	\item $\hatv$: Last seen open state version.
 	\item $\hats$: Sequence number of latest seen snapshot.
-	\item $\hatSigma \in {(\tyNatural \times \tyBytes)}^{*}$: Accumulator of
-	      signatures of the latest seen snapshot, indexed by parties.
+	\item $\hatSigma \in {(\tyNatural \times \tyBytes)}^{*}$:
+		Accumulator of signatures of the latest seen snapshot, indexed by parties.
 	\item $\hatmL$: UTxO set representing the local ledger state resulting from
-	      applying $\hatmT$ to $\bar{S}.U$ to validate requested transactions.
+		applying $\hatmT$ to $\bar{S}.U$ to validate requested transactions.
 	\item $\hatmT \in \mT^{*}$: List of transactions applied locally and pending
-	      inclusion in a snapshot (if this party is the next leader).
-	\item $\tx_{\omega} \in \mathcal{T}^{?}$: Pending decommit transaction, whose outputs are to be decommitted from the head. May be $\bot$ if nothing to decommit.
+		inclusion in a snapshot (if this party is the next leader).
+	\item \red{$U_\alpha \in \gcUTXOset$: Pending increment UTxO to be deposited to the head.}
+	\item $U_\omega \in \gcUTXOset$: Pending decrement UTxO to be withdrawn from the head.
+	\item \red{$U_\theta \in U_\alpha \cup U_\omega \cup \bot$: Either a decrement UTxO, an increment UTxO or $\bot$.
+		May be $\bot$ if nothing is pending to be applied.}
+	\item \red{$\theta \in Normal \cup Inc \cup Dec$: A type of a snapshot object.}
```

Comment:
Instead of defining $U_\alpha$ and $U_\omega$, you should unify the types directly (otherwise: what's the point of having a unified $U_\theta$?) 

Maybe using something like:

$U_\theta \in \mathsf{Normal} \cup (\mathsf{Inc}, \mathsf{UTxOs}) \cup (\mathsf{Dec}, \mathsf{UTxOs})$


(We should really define $\mathsf{UTxOs}$ better)

Reply:
Also, I think you could avoid the dedicated $\theta$ and just define that $\eta_\theta$ includes that discriminator?